src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -984,34 +984,76 @@ def is_valid_uint_value(value, bit_count=64):
             return False
         return value < 2**bit_count
 
+    @staticmethod
+    def is_valid_int_value(value, bit_count=8):
+        """
+        Checks if 'value' is a signed integer that fits into 'bit_count' bits.
+        For example, for int8: -128 <= value <= 127.
+        """
+        min_val = -(2 ** (bit_count - 1))
+        max_val = (2 ** (bit_count - 1)) - 1
+        return isinstance(value, int) and (min_val <= value <= max_val)
+
+    @staticmethod
+    def is_valid_bool_value(value):
+        return isinstance(value, bool)
+
+    @staticmethod
+    def is_valid_octstr(value):
+        return isinstance(value, bytes) and len(value) > 0
+
     @staticmethod
     def is_valid_str_value(value):
         return isinstance(value, str) and len(value) > 0
 
     def assert_valid_uint64(self, value, field_name):
-        """Asserts that the value is a valid uint64."""
         asserts.assert_true(self.is_valid_uint_value(value, bit_count=64),
-                            f"{field_name} should be a uint64 or NULL.")
+                            f"{field_name} should be a uint64.")
 
     def assert_valid_uint32(self, value, field_name):
-        """Asserts that the value is a valid uint32."""
         asserts.assert_true(self.is_valid_uint_value(value, bit_count=32),
-                            f"{field_name} should be a uint32 or NULL.")
+                            f"{field_name} should be a uint32.")
 
     def assert_valid_uint16(self, value, field_name):
-        """Asserts that the value is a valid uint16."""
         asserts.assert_true(self.is_valid_uint_value(value, bit_count=16),
-                            f"{field_name} should be a uint16 or NULL.")
+                            f"{field_name} should be a uint16.")
 
     def assert_valid_uint8(self, value, field_name):
-        """Asserts that the value is a valid uint16."""
         asserts.assert_true(self.is_valid_uint_value(value, bit_count=8),
-                            f"{field_name} should be a uint8 or NULL.")
+                            f"{field_name} should be a uint8.")
+
+    def assert_valid_int64(self, value, field_name):
+        asserts.assert_true(self.is_valid_int_value(value, bit_count=64),
```

Comment:
should probably convert these (and extend in some way maybe) https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/matter_testing_infrastructure/chip/testing/matter_asserts.py#L46 (i.e. `assert_int_in_range`)

Reply:
The asserts have been removed in matter_testing.py , and use the asserts from matter_asserts